client/my-sites/invites/invite-accept-logged-in/index.jsx

```diff
@@ -21,6 +22,14 @@ export default React.createClass( {
 
 	displayName: 'InviteAcceptLoggedIn',
 
+	acceptInvite() {
```

Comment:
perhaps change this method name to simply `accept` 
seeing acceptInvite written so close together feels a little confusing.